packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -330,6 +331,7 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
   late Map<int, Offset> _pointerLocations;
   late List<int> _pointerQueue; // A queue to sort pointers in order of entrance
   final Map<int, VelocityTracker> _velocityTrackers = <int, VelocityTracker>{};
+  late Offset _delta;
```

Comment:
Do we need an instance variable? In `_update` we always need to recompute the value so maybe we can use a local variable instead?

Reply:
Can you take another look at this?  Sorry, I was in the middle of changing the PR, I should have used a WIP label.  As it is now, I need the previousFocalPoint to calculate delta (since now it's just the delta since the last call, not since the start).  So I think I either need `_delta` or `_previousFocalPoint` as an instance variable.

Reply:
Ah sorry I thought `ScaleUpdateDetails` was constructed in `_update` nvm.